librz/bin/p/bin_coff.c

```diff
@@ -9,6 +9,7 @@
 #include <rz_util/ht_uu.h>
 
 #include "coff/coff.h"
+#include "rz_vector.h"
```

Comment:
```suggestion
#include <rz_vector.h>
```

Reply:
Maybe this header file is not needed to declare here. It was added by my neovim accidentally. I will remove it later.